spec/models/redirection_spec.rb

```diff
@@ -21,6 +21,17 @@
     end
   end
 
+  describe ".in_ring_order" do
+    it "returns redirections in the order they appear next in the ring" do
+      create(:redirection)
+      first = Redirection.order(id: :asc).first
```

Comment:
True, but I think it shows some intent here. I want the record with the lowest numbered ID rather than "whatever the first one by some metric" is.